content/reference.html

```diff
@@ -1,1710 +1,7 @@
-          <tr>
-            <!--  -->
-            <td width = "90%">
-              <p><code><a href="stream_watermark">stream_watermark()</a></code> </p>
-            </td>
-            <td><p>Watermark Stream</p></td>
-          </tr>
-          <tr>
-            <!--  -->
-            <td width = "90%">
-              <p><code><a href="stream_write_console">stream_write_console()</a></code> </p>
-            </td>
-            <td><p>Write Console Stream</p></td>
-          </tr>
-          <tr>
-            <!--  -->
-            <td width = "90%">
-              <p><code><a href="stream_write_csv">stream_write_csv()</a></code> </p>
-            </td>
-            <td><p>Write CSV Stream</p></td>
-          </tr>
-          <tr>
-            <!--  -->
-            <td width = "90%">
-              <p><code><a href="stream_write_delta">stream_write_delta()</a></code> </p>
-            </td>
-            <td><p>Write Delta Stream</p></td>
-          </tr>
-          <tr>
-            <!--  -->
-            <td width = "90%">
-              <p><code><a href="stream_write_json">stream_write_json()</a></code> </p>
-            </td>
-            <td><p>Write JSON Stream</p></td>
-          </tr>
-          <tr>
-            <!--  -->
-            <td width = "90%">
-              <p><code><a href="stream_write_kafka">stream_write_kafka()</a></code> </p>
-            </td>
-            <td><p>Write Kafka Stream</p></td>
-          </tr>
-          <tr>
-            <!--  -->
-            <td width = "90%">
-              <p><code><a href="stream_write_memory">stream_write_memory()</a></code> </p>
-            </td>
-            <td><p>Write Memory Stream</p></td>
-          </tr>
-          <tr>
-            <!--  -->
-            <td width = "90%">
-              <p><code><a href="stream_write_orc">stream_write_orc()</a></code> </p>
-            </td>
-            <td><p>Write a ORC Stream</p></td>
-          </tr>
-          <tr>
-            <!--  -->
-            <td width = "90%">
-              <p><code><a href="stream_write_parquet">stream_write_parquet()</a></code> </p>
-            </td>
-            <td><p>Write Parquet Stream</p></td>
-          </tr>
-          <tr>
-            <!--  -->
-            <td width = "90%">
-              <p><code><a href="stream_write_text">stream_write_text()</a></code> </p>
-            </td>
-            <td><p>Write Text Stream</p></td>
-          </tr>
-          <tr>
-            <!--  -->
-            <td width = "90%">
-              <p><code><a href="reactiveSpark">reactiveSpark()</a></code> </p>
-            </td>
-            <td><p>Reactive spark reader</p></td>
-          </tr>
-      </tbody>
-      </table>
-
-  </div>
-
-  </div>
-
-
-
-
-
-
+<iframe src="reference_home.html" title="" frameborder="0" scrolling="no" onload="this.style.height=(this.contentWindow.document.body.scrollHeight + 5)+'px';this.style.width=(this.contentWindow.document.body.scrollWidth + 5)+'px';"></iframe>
```

Comment:
@javierluraschi This actually looks great, as long as no one clicks "view page source" to see my ugly inline javascript code here :joy: 

With this configuration, no one will need to fiddle with the pkgdown template files in sparklyr repo to see whether the rendered output makes sense for spark.rstudio.com. So, I think this is exactly the type of decoupling we needed to make our lives easier.